tests/onegov/fsi/test_upgrade.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from onegov.fsi.upgrade import parse_and_split_address_field
+
+
+@pytest.mark.parametrize('address,expected', [
```

Comment:
Does parametrizing increase the test count? Because this will imbalance the CI by adding many new and fast tests...

Reply:
I made this to ensure all addresses currently in /onegov_agency/bs, zg and ar will properly migrated to the new address scheme

Reply:
What I mean is: Why parametrizing the test and not just use the list of (address, expected) tuples within a single test? I suspect that parametetrized tests are counted as individual tests and therefore influence to test splitting in the CI (I might be wrong). This will lead to adding 20 new and very fast tests, which will probably been executed in one of the four processes (since test splitting is done by dividing to the number of processes). This one CI process will then have a shorter runtime compared to the other ones (and probably increasing the runtime at least one of the others and therefore the whole pipeline).

Reply:
Will adjust it